File: java-gradle/plugins/jdbc/src/gatling/java/{{ .PackagePath }}/{{ .NameWord }}/scenarios/JdbcScenario.java
package {{ .Package }}.{{ .NameWord }}.scenarios;

import io.gatling.javaapi.core.ScenarioBuilder;
import {{ .Package }}.{{ .NameWord }}.cases.JdbcActions;

import static io.gatling.javaapi.core.CoreDsl.scenario;

public final class JdbcScenario {

    private JdbcScenario() {
    }

    public static ScenarioBuilder create() {
        return scenario("Jdbc Scenario")
                .exec(JdbcActions.createTable);
    }
}

File: kotlin-maven/plugins/jdbc/src/test/kotlin/{{ .PackagePath }}/{{ .NameWord }}/scenarios/JdbcScenario.kt
package {{ .Package }}.{{ .NameWord }}.scenarios

import io.gatling.javaapi.core.CoreDsl.scenario
import io.gatling.javaapi.core.ScenarioBuilder
import {{ .Package }}.{{ .NameWord }}.cases.JdbcActions

object JdbcScenario {
    fun create(): ScenarioBuilder =
        scenario("Jdbc Scenario")
            .exec(JdbcActions.createTable)
}
